src/viam/sdk/services/mlmodel/server.cpp

```diff
@@ -185,10 +193,16 @@ ::grpc::Status MLModelServiceServer::Metadata(
     };
 
     auto status = pack_tensor_info(*metadata_pb.mutable_input_info(), md.inputs);
-    if (!status.ok())
+    if (!status.ok()) {
```

Comment:
placate tidy. Personally, I dislike requiring braces for single line conditionals. The argument in favor of it is that it prevents logic errors where a statement is invisibly guarded due to bad indenting:

```
if (x)
statement;
```

Because the indenting is wrong it might be possible to not see that `statement` is guarded. But in the age of pretty-printers that would get re-indented so I don't think it is important.

If others feel similarly maybe we can update the tidy rules to allow non-braced single line conditionals.

Reply:
I don't feel strongly about one syntax being preferable, and agree that there isn't a strong argument to disallow non-braced single line conditionals. I am happy to allow them going forward.

Reply:
If it bites me again I'll fold a suppression of this rule into our clang-tidy file in some subsequent PR. Otherwise I won't bother.